.byebug_history

```diff
@@ -0,0 +1,54 @@
+exit
+subject.deadable?
+subject.dead?
+subject.deadable
+subject
+worker.deadable
```

Comment:
blindly including all files into a commit is bad practice

i recommend you add a global gitignore to your user
```
$ cat ~/.gitconfig 
[core]
        excludesfile = ~/.gitignore_global
```
and add files such as `.byebug_history` in `~/.gitignore_global`